src/Tizen.Applications.Common/Tizen.Applications/Event.cs

```diff
@@ -0,0 +1,134 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using static Interop.Event;
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// The Event API provides functions to receive events.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class Event : IDisposable
+    {
+        internal IntPtr _handler;
+        private readonly string _eventName = null;
+        private Interop.Event.EventCallback _eventCallBack;
+
+        /// <summary>
+        /// Initializes the instance of the Event class.
+        /// </summary>
+        /// <param name="eventName">The name of the event.</param>
+        /// <exception cref="System.ArgumentException">Thrown when eventName is null or empty.</exception>
+        /// <example>
+        /// <code>
+        /// Event ev = new Event("tizen.system.event.usb_status");
+        /// ev += EventReceivedCallback;
+        /// </code>
+        /// </example>
+        /// <since_tizen> 6 </since_tizen>
+        public Event(string eventName)
```

Comment:
It is too poor usability to use. Developer hard to know which eventName is available on this parameter.
Please check AppControlOperations class

Reply:
I'll make another PR for Event.
For now, i need to discuss design for Event with my coworkers.
Thank you for your code review.